src/trust_is_risk.js

```diff
@@ -14,10 +14,46 @@ var TrustDB = require("./trust_db");
 var DirectTrust = require("./direct_trust");
 
 class TrustIsRisk {
-  node : bcoin$FullNode
+  node : (bcoin$FullNode | bcoin$SPVNode)
   db : TrustDB
 
-  constructor(node : bcoin$FullNode) {
+  fakePubKeyArray : Array<number>
+  fakePubKey : Buffer
+  fakeKeyRing : KeyRing
+  tag : Buffer
+
+  compressedFakePubKeyArray : Array<number>
+  compressedFakePubKey : Buffer
+  compressedFakeKeyRing : KeyRing
+  compressedTag : Buffer
+
+  constructor(node : (bcoin$FullNode | bcoin$SPVNode)) {
+    this.fakePubKeyArray = [0x04,            // constant 0x04 prefix
+      0x54, 0x72, 0x75, 0x73, 0x74, 0x20, 0x69, 0x73,
+      0x20, 0x52, 0x69, 0x73, 0x6b, 0x00, 0x00, 0x00,    // 32 bytes with the x coordinate
+      0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,    // containing ASCII "Trust is Risk"
```

Comment:
Deduce this from the string 'Trust is Risk' instead of typing it out fully; at least the x part

Reply:
Removed compressed versions completely